PluginGemfile

```diff
@@ -1,9 +1,13 @@
+source 'https://rubygems.org'
```

Comment:
Why was this removed in the first place? @schmijos knows maybe?

Reply:
`PluginGemfile` is merged into the Redmine gemfile. Adding this line again leads to duplicate sources, doesn't it?

Reply:
Ah I see. Should we use a separate `Gemfile` and `Gemfile.lock`? You did delete both in https://github.com/renuo/redmine_tracky/commit/c7b74a99fabcb5af5459ba480b5edf9d9f1aa550